### PR TITLE
some data/ links

### PR DESCRIPTION
**Pull request by darkblue-b**
links to the data/ dir on beta4

### CODE REVIEW

File: bin/install_R.sh [hunk 1]
```diff
@@ -153,17 +153,15 @@ chown "$USER_NAME.$USER_NAME" "$USER_HOME/Desktop/r.desktop"
...
-# mkdir -p /usr/local/share/data/vector
-# ln -s /usr/lib/R/site-library/rgdal/vectors \
-#    /usr/local/share/data/vector/R
-
 ### install R package containing sids.shp needed for postgis quickstart
 Rscript -e "install.packages('spData')"
 
+# link sample data to central location
+ln -s /usr/local/lib/R/site-library/spData/shapes \
+       $USER_HOME/data/spdata_shapes
```
Comment (kalxas) [inline bin/install_R.sh]: Please add the link to the shared folder, not the user folder.
/usr/local/share/data/vector/R

File: bin/install_rasdaman.sh [hunk 2]
```diff
@@ -236,6 +236,9 @@ service tomcat8 stop
 ##---- disk space v12
 rm -rf /opt/rasdaman/share/rasdaman/doc/doc-guides  ## 3.3MB
 
+chmod a+r /var/lib/tomcat8/webapps/52nWPS/testData/elev_srtm_30m21.tif
+ln -s /var/lib/tomcat8/webapps/52nWPS/testData/elev_srtm_30m21.tif \
+       $USER_HOME/data/raster/elev_srtm_30m21.tif
```
Comment (kalxas) [inline bin/install_rasdaman.sh]: This should not be here, this is the rasdaman installer.

File: bin/load_gisdata.sh [hunk 3]
```diff
@@ -234,6 +234,15 @@ cd nc_data
 
 mkdir -p "$DATA_FOLDER/north_carolina"
 
+
+##--
+ln -s /usr/lib/python2.7/dist-packages/gisdata/data/good/raster/relief_san_andres.tif \
+       $USER_HOME/data/raster/relief_san_andres.tif
```
Comment (kalxas) [inline bin/load_gisdata.sh]: Same here, please link to the shared folder, not specifically for one user.

**Comment by darkblue-b** [comment]
revision ready

**Pull request merged**